File: app/src/main/java/com/example/voxelvisage/api/FaceModel.java
package com.example.voxelvisage.api;

import com.google.gson.annotations.SerializedName;
import java.util.List;

public class FaceModel {

    @SerializedName("originalImage")
    private String originalImage;

    @SerializedName("imageWithLandmarks")
    private String imageWithLandmarks;

    @SerializedName("landmarksCoordinates")
    private List<List<Integer>> landmarksCoordinates;

    @SerializedName("faceCoordinates")
    private List<Integer> faceCoordinates;

    public String getOriginalImage() {
        return originalImage;
    }

    public String getImageWithLandmarks() {
        return imageWithLandmarks;
    }

    public List<List<Integer>> getLandmarksCoordinates() {
        return landmarksCoordinates;
    }

    public List<Integer> getFaceCoordinates() {
        return faceCoordinates;
    }
}

File: FaceModel.py
from typing import List, Tuple
from pydantic import BaseModel

class FaceModel(BaseModel):
    originalImage: str
    imageWithLandmarks: str
    landmarksCoordinates: List[Tuple[int, int]]
    faceCoordinates: Tuple[int, int, int, int]

    class Config:
        arbitrary_types_allowed = True